primitives/src/constants.rs

```diff
@@ -66,7 +66,7 @@ parameter_types! {
     pub const AdvisoryBond: Balance = 25 * CENT;
     pub const DisputeBond: Balance = 5 * BASE;
     pub const DisputeFactor: Balance = 2 * BASE;
-    pub const DisputePeriod: BlockNumber = BLOCKS_PER_DAY;
+    pub const DisputePeriod: BlockNumber = 2 * BLOCKS_PER_DAY;
```

Comment:
This is a suggestion to have a longer period (2 days), that everybody on the planet earth has enough time to vote on a dispute.